notes.bash_completion

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+_notes_complete_notes() {
+    local configured_dir=${NOTES_DIRECTORY%/} # Remove trailing slashes
+    local notes_dir="${configured_dir:-$HOME/notes}"
+    local items=($(compgen -f "$notes_dir/$1"))
+    for item in "${items[@]}"; do
+        [[ -d $item ]] && item="$item/"
+        local filename=${item#$notes_dir/}
+        COMPREPLY+=("${filename%.md}")
+    done
+}
+
+_notes()
+{
+    local cur
+    readonly commands="new find grep open"
```

Comment:
When using this on my machine, I get lots of `bash: commands: readonly variable` when trying to use completion, which I assume is caused by this line. Completion still works, other than that spurious extra output. Using bash 4.4.5.

Any idea why? I'd like to fix this before merging.